packages/medusa-payment-paypal/src/services/paypal-provider.ts

```diff
@@ -0,0 +1,386 @@
+import PayPal from "@paypal/checkout-server-sdk"
```

Comment:
thought(non-blocking): It seems this SDK has been deprecated in favor of a direct Rest API integration. Would it make sense to use that instead? I believe the flows are identical. The most significant change is that we need to create our own Axios client.